.github/workflows/python-app.yml

```diff
@@ -30,4 +30,8 @@ jobs:
         mypy --ignore-missing-imports --disallow-untyped-defs --no-strict-optional --config-file setup.cfg
     - name: Unit tests
       run: |
-        pytest
\ No newline at end of file
+        pytest
+    - name: Integration tests
+      run: |
+        source ./tests/scripts/initialize_database.sh
+        source ./tests/scripts/elasticsearchstore_integration_test.sh
```

Comment:
Please create another job for integration test, here everything run in the job 'build', while we only want linter and type checking in this build. Create also another job for unit tests.